internal/namespaces/instance/v1/custom_server_test.go

```diff
@@ -279,6 +279,49 @@ func Test_ServerDelete(t *testing.T) {
 	interactive.IsInteractive = false
 }
 
+// These tests needs to be run in sequence
+// since they are using the interactive print
+func Test_ServerTerminate(t *testing.T) {
+	interactive.IsInteractive = true
+
+	t.Run("without IPs", core.Test(&core.TestConfig{
+		Commands:   GetCommands(),
+		BeforeFunc: core.ExecStoreBeforeCmd("Server", "scw instance server create image=ubuntu-bionic -w"),
+		Cmd:        `scw instance server terminate {{ .Server.ID }}`,
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(0),
+		),
```

Comment:
We should add a CheckFunc that test that IP still exist by doing a `GetIP` on the API